Sources/Containerization/NATInterface.swift

```diff
@@ -18,6 +18,7 @@ public struct NATInterface: Interface {
     public var address: String
     public var gateway: String?
     public var macAddress: String?
+    public var routes: [Route] { [] }
 
     public init(address: String, gateway: String?, macAddress: String? = nil) {
```

Comment:
Leave the type defined as `public var routes: [Route]` but just add it to the constructor with a default value of []